CMakeLists.txt

```diff
@@ -208,12 +218,17 @@ add_custom_command(
 #----------------------------------------------------------------------
 # Add JSON parser
 #----------------------------------------------------------------------
```

Comment:
We should probably update this comment to indicate gtest is added here now too.

Reply:
Will fix.